assam_budget_board/cms_plugins.py

```diff
@@ -41,13 +41,11 @@ def render(self, context, instance, placeholder):
         context = super(GrantSummaryPlugin, self).render(context, instance, placeholder)
         context.update({
         	"fiscalYear" : {
-        		"be" : "Budget 2018-19", 
-        		"actuals" : "Actuals 2016-17",
-				"bePrev" : "Budget 2017-18",
-				"re" : "Revised 2017-18"
+                "be" : "BUDGET_%s_%s" % (settings.FISCAL_YEAR, (settings.FISCAL_YEAR+1)%1000), 
+                "bePrev" : "BUDGET_%s_%s" % (settings.FISCAL_YEAR-1, (settings.FISCAL_YEAR)%1000)
         		}
         	})
-
+        print("BUDGET_%s_%s" % (settings.FISCAL_YEAR, (settings.FISCAL_YEAR+1)%1000))
```

Comment:
@shreyaagrawal0809 Can we remove print from here?

Reply:
Done